src/data.js

```diff
@@ -12,210 +12,95 @@
 */
 
 export const quizData = {
-    currentQuestionIndex: 0,
```

Comment:
As I mention, try to remove any change from this file, so later we don't have conflict when merging. It's fine to have it locally, just not commit it.